### PR TITLE
Add gradient mode for animated dyes

### PR DESCRIPTION
**Pull request by NopoTheGamer**
https://discord.com/channels/516977525906341928/850059600182509619/1264440171978555422

image

also adds vanilla dyes at the bottom of the hypixel dyes menu

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/ItemCustomizationUtils.java [hunk 1]
```diff
@@ -86,7 +86,10 @@ public class ItemCustomizationUtils {
 
 	public static List<String> speedGuide = Lists.newArrayList(
 		EnumChatFormatting.AQUA + "This is how fast the dyes will cycle, in ticks",
-		EnumChatFormatting.GRAY + "Hypixel dyes cycle every 2 ticks"
+		EnumChatFormatting.GRAY + "§6Hypixel §7dyes cycle every 2 ticks",
+		EnumChatFormatting.GRAY + "",
+		EnumChatFormatting.GRAY + "In the §dgradient mode §7this is how many colours between colours",
```
Comment (nea89o) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/ItemCustomizationUtils.java:91]:
```suggestion
		EnumChatFormatting.GRAY + "In the §dgradient mode §7this decides the amount of intermediary colours",
```

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/DyeMode.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.moulberry.notenoughupdates.miscgui.itemcustomization
+
+enum class DyeMode {
+    ANIMATED,
```
Comment (nea89o) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/DyeMode.kt:23]: Arent both modes animated? Some other names could be harsh, sharp, clean or banding.

Comment (NopoTheGamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/DyeMode.kt:23]: Thoughts on "Cycling" instead?

Comment (Lulonaut) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/DyeMode.kt:23]: 🚴‍♂️

Comment (Madeleaan) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/DyeMode.kt]: 🚴‍♂️

**Review by Lulonaut — APPROVED** [review]

**Review by nea89o — APPROVED** [review]

**Pull request merged**